splinterd/sample_configs/splinterd.toml.example

```diff
@@ -13,40 +13,95 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# The default values are shown for each setting, unless otherwise noted.
+
 # Config file format version
 version = "1"
 
+# Sets a new ID for the node. The node ID must be unique across the
+# network (for all Splinter nodes that could participate on the same circuit).
+# (default 'nXXXXX', where 'XXXXX' is 5 random numbers)
 node_id = "012"
 
+# Human-readable name for the node
+# (default 'Node {node_id}')
+display_name = "node-012"
+
 # Endpoint used for service to daemon communication.
+# (default "127.0.0.1:8043")
 service_endpoint = "127.0.0.1:8043"
 
 # Endpoints used for daemon to daemon communication.
+# (default ["protocol_prefix://127.0.0.1:8044"])
 network_endpoints = ["tcps://127.0.0.1:8044"]
 
+# Endpoint used for REST API communication
+# (default "127.0.0.1:8080")
+bind = "127.0.0.1:8080"
+
 # A list of splinter nodes the daemon will automatically
 # connect to on start up.
+# (default []; empty list)
 peers = []
 
 # The type of storage that should be used to store circuit state. Option are
-# currently "yaml" or "memory"
+# currently "yaml" or "memory".
+# (default "yaml")
 storage = "yaml"
 
 # List of certificate authority certificates (*.pem files).
-tls_ca_file = "certs/ca.pem"
+# (default "/etc/splinter/ca.pem")
```

Comment:
should be ` # (default "/etc/splinter/certs/ca.pem")`

Reply:
Updated all